docu/Concepts/Snippets/1-Blockchain/Connector/Dezentralität.md

```diff
@@ -0,0 +1,70 @@
+## Dezentralität
+
+Ich habe von Anfang an versucht Gradido so dezentral wie möglich zu entwerfen.
+Dezentral an der Basis. Kryptografie und Blockchain-Technologie sind meiner Meinung nach die besten Technologien dafür.
+Sie ermöglichen einen Ansatz bei dem der Benutzer und seine Daten im Mittelpunkt stehen. In der Form,
+das nur der Benutzer selbst die volle Kontrolle über seine Daten haben kann. 
+Dadurch wird der Benutzer unabhägig von Server-Admins und das Gesamtsystem hat weniger Points of Failure.
+Bei den Crypto-Coins wie Bitcoin wird diese Kontrolle des Benutzers über seine Daten durch die Nutzung
+von Public-Private Kryptografie erreicht. Damit sind zwei grundlegende Arten von Operationen möglich.
+
+Die eine Art der Operationen kann nur derjenige ausführen, der Zugriff auf den Private Key hat. 
+Das Ergebnis kann jedoch von allen überprüft werden die Zugriff auf den Public Key haben. 
+
+Und es gibt Operationen die man mit dem public key ausführen kann, bei denen nur der Inhaber das Private Keys das 
+Ergebnis sehen kann. 
+
+Wenn mehr Wert auf Anonymität gelegt wird, gibt es auch sogenannte Zero-Knowledge Protokolle wie bei zcash welche Transaktionen ermöglichen 
+die nicht nachvollziehbar sind. 
```

Comment:
Is this applicable to our solution? I claim that we cannot make a 100% cryptographical proof if a transaction is correct, since we cannot map all rules of gradido to math. Can the zcash protocol be applied to our solution while providing anonymity, security and tamper proof?

Furthermore some formating:
```suggestion
Wenn mehr Wert auf Anonymität gelegt wird, gibt es auch sogenannte Zero-Knowledge Protokolle wie bei zcash welche Transaktionen ermöglichen die nicht nachvollziehbar sind. 
```